docs/content/readme.md

```diff
@@ -58,25 +55,25 @@ Instantly get access to our documents and meeting invites [http://kubestellar.io
 
 - The [`#kubestellar-dev` channel](https://cloud-native.slack.com/archives/C097094RZ3M) in the [CNCF Slack workspace](https://communityinviter.com/apps/cloud-native/cncf)
 - Our mailing lists:
-    - [kubestellar-dev](https://groups.google.com/g/kubestellar-dev) for development discussions
-    - [kubestellar-users](https://groups.google.com/g/kubestellar-users) for discussions among users and potential users
+  - [kubestellar-dev](https://groups.google.com/g/kubestellar-dev) for development discussions
+  - [kubestellar-users](https://groups.google.com/g/kubestellar-users) for discussions among users and potential users
```

Comment:
We need indenting by four spaces because that is what mkdocs recognizes as making a nested paragraph.